sploitkit/core/components/files.py

```diff
@@ -16,8 +17,8 @@ class FilesManager(dict):
     def _file(self, locator, *args, **kwargs):
         """ Simple local file copier. """
         with open(locator.split("://", 1)[1], 'rb') as f:
-            self[url] = f.read()  #FIXME: use file descriptor
-    
+            self[url] = f.read()  # FIXME: use file descriptor
```

Comment:
Note that, as a general rule, I write _permanent comments_ using the white space after the hashtag, what I don't do in the case of a _temporary comment_ aimed to test something or to mark it as a TODO/FIXME. This is easier to search for in a classical text editor.